recipes/syft/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set goname = "syft" %}
+{% set version = "0.92.0" %}
+
+
+package:
+  name: {{ goname|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  - folder: src
+    url: https://github.com/anchore/syft/archive/v{{ version }}.tar.gz
+    sha256: 15073d2a349ab2ebcba7e4f446016fdb6f9422b94bd59bcb8dd7839fab2e4807
+  - url: https://raw.githubusercontent.com/xi2/xz/48954b6210f8d154cb5f8484d3a3e1f83489309e/LICENSE
+    sha256: 5c640056d513f276d56635af7a0bed81cb7eac02a31b6e56d40c416557a8d8ce
+    fn: LICENSE-xz
```

Comment:
This package has a non-standard license:
https://github.com/xi2/xz/blob/master/LICENSE

How problematic is that?

Reply:
It's a public domain declaration. This is fine.